common/src/main/java/com/wynntils/screens/activities/WynntilsContentBookScreen.java

```diff
@@ -331,7 +331,17 @@ public void reloadContentBookWidgets(boolean resetPage) {
             trackedActivity = newTrackedActivity;
 
             if (trackedActivity.description().isPresent()) {
-                trackedDescription = trackedActivity.description().get();
+                trackedDescription = trackedActivity.description().get().map(part -> {
+                    if (part.getPartStyle().getColor().equals(CustomColor.fromChatFormatting(ChatFormatting.GRAY))) {
+                        return part.withStyle(partStyle -> partStyle.withColor(CommonColors.WHITE));
+                    } else if (part.getPartStyle()
+                            .getColor()
+                            .equals(CustomColor.fromChatFormatting(ChatFormatting.WHITE))) {
+                        return part.withStyle(partStyle -> partStyle.withColor(CommonColors.AQUA));
+                    } else {
+                        return part;
+                    }
+                });
```

Comment:
Isn't this same exact code present in the overlay? If so, there would be wise to have a common method.